src/commands/setup.rs

```diff
@@ -29,7 +33,66 @@ impl Setup {
             Err(e) => panic!("{}", e),
         };
 
+        // Iterate networks that will require iptables rules
+        for (_, v) in _network_options.network_info {
+            let network_name = v.network_interface.unwrap();
+            // for nets in v.subnets {
+            if let Some(nets) = v.subnets{
+                self.setup_iptables(&network_name, nets);
+            }
+        }
         debug!("{:?}", "Setup complete");
-        //()
+    }
+    /// Set up required iptables for NAT
+    ///
+    ///
+    pub fn setup_iptables(&self, network_name: &str, subnets: Vec<Subnet>) {
```

Comment:
I think can have this function in `network` create not at the frontend.

Reply:
can you be more exact. point to exactly where

Reply:
@baude we could create a new file `src/network/core.rs` maybe like this ->  https://github.com/containers/netavark/pull/17